libs/sdk-ui-dashboard/src/presentation/dragAndDrop/draggableWidget/useAddInitialSectionHandler.ts

```diff
@@ -1,70 +0,0 @@
-// (C) 2022-2024 GoodData Corporation
```

Comment:
This file was not used anywhere. Probably was not even exposed via barrel file. It should be safe to remove 🤞